src/date-range-picker/calendar/grids/index.tsx

```diff
@@ -117,25 +115,24 @@ export const Grids = ({
         if (!focusedDate || !isDateEnabled(focusedDate)) {
           return;
         }
-
+        // onFocusDate(null);
```

Comment:
What happened to this line?

Reply:
There is no onFocusDate() currently on [the page ](https://github.com/cloudscape-design/components/blob/main/src/date-range-picker/calendar/grids/index.tsx). That is an artifact when I experiments using the `useCalendarGridKeyboardNavigation` hook from [calendar](https://github.com/cloudscape-design/components/blob/85cb377f31b2b184b1c26a4a1c320c6799ca9716/src/calendar/grid/use-calendar-grid-keyboard-navigation.ts#L20) .  I will cleanup.